openvidu-browser/src/OpenVidu/Stream.ts

```diff
@@ -217,6 +217,8 @@ export class Stream extends EventDispatcher {
         super();
 
         this.session = session;
+        this.audioActive = false;
```

Comment:
This initialization is unnecesary because of the boolean variables has `false` by default.
Please, remove it.